### PR TITLE
pipeline to publish npm package

### PR DESCRIPTION
**Pull request by 0xxlegolas**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - uses: actions/setup-node@v3
+    with:
+      node-version: 16
+      registry-url: 'https://registry.npmjs.org'
+    - run: npm install
+    - run: |
+        npm prepack
+        npm publish 
+      env: 
+        GITHUB_WORKSPACE: ${GITHUB_WORKSPACE}
```
Comment (iv0rish) [inline .github/workflows/release.yml]: @toniya-klaytn Why do you need this one?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (iv0rish) [inline .github/workflows/release.yml]:
```suggestion
    - name: Publish package to npm.js
      uses: JS-DevTools/npm-publish@v1
      with:
        token: ${{ secrets.NPM_TOKEN }}
        access: public
```

DevOps is using this component to build & publish js packages to npm, please consider replacing publish step to this one.

Comment (0xxlegolas) [reply .github/workflows/release.yml]: sure tony will replace

File: .github/workflows/release.yml (hunk 1 above)
Comment (0xxlegolas) [reply .github/workflows/release.yml]: npm prepack is to build the library

**Comment by 0xxlegolas** [comment]
@iv0rish  Could you review again ?

**Review by iv0rish — APPROVED** [review]

**Pull request merged**